tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/jms/SimpleJNDIClientTest.java

```diff
@@ -115,6 +115,18 @@ public void testEmptyConnectionFactoryString() throws NamingException, JMSExcept
 
    }
 
+   @Test
+   public void testConnectionFactoryStringWithInvalidParameter() throws NamingException, JMSException {
+      Hashtable<String, String> props = new Hashtable<>();
```

Comment:
Are you asserting anything here? you could use the AssertionLogger (whatever the name that is)

Reply:
I added the assertion using the AssertionLogger.